packages/vscode-extension/src/debugging/DebugSession.ts

```diff
@@ -15,6 +15,8 @@ const VSCODE_JS_DEBUGGER_TYPE = "com.swmansion.proxy-debugger";
 export const DEBUG_CONSOLE_LOG = "RNIDE_consoleLog";
 export const DEBUG_PAUSED = "RNIDE_paused";
 export const DEBUG_RESUMED = "RNIDE_continued";
+export const SCRIPT_PARSED = "RNIDE_bundleParsed";
```

Comment:
I saw this already in the previous PR but was already merged before I can comment. Would prefer to not use "bundle" for this event as it actually maps 1-to-1 to script parsed event. In particular this gets triggered for individual scripts that we load via debugger while bundle has a more concrete meaning. A script could be the main bundle but not everything that triggers scriptParsed is a bundle.

To further deepen the confusion here we actually use "bundle parsed" for the string while the constant is called "script parsed"

Reply:
I agree.